ykview/src/main.rs

```diff
@@ -0,0 +1,59 @@
+use std::io::Read;
+
+fn main() {
+    let mut args = std::env::args();
```

Comment:
Can you add some code which prints the usage in case the wrong arguments are used?

Reply:
Done